docs/specifications/tests/DNSSEC-TP/dnssec02.md

```diff
@@ -162,52 +169,78 @@ queries follow, unless otherwise specified below, what is specified for
              the DNSKEY.
           2. Else, if the DS values (algorithm and digest) do not match the
              DNSKEY record then add DS key tag and name server IP to the
-             *No match DS DNSKEY* set.
-       4. If bit 7 of the DNSKEY flags field is unset (value 0), then add DS key
-          tag and name server IP to the *DNSKEY not for zone signing* set.
+             *No Match DS DNSKEY* set.
+       4. If bit 7 of the DNSKEY flags field is unset (value 0), then do:
+          1. Add DS key tag and name server IP to the
+             *DNSKEY Not for Zone Signing* set.
+          2. Go to next DS.
        5. If bit 15 of the DNSKEY flags field is unset (value 0), then add the
           DNSKEY record key tag and name server IP to the *DNSKEY not SEP*
           set.
-       6. Find the equivalent RRSIG in *DNSKEY RRSIG* by key ID (key tag). If
-          there is more than one such RRSIG record, select the correct one.
-       7. If matching RRSIG is not found, add DNSKEY record key tag and name
-          server IP to the *No matching DNSKEY RRSIG* set.
-       8. Else, if the Zonemaster installation does not have support for the
+       6. Add the DNSKEY record, key tag, name server IP to the *DNSKEY Match DS*
+          set.
+    7. Empty the *Has DNSKEY RRSIG Match DS* set.
```

Comment:
This isn't needed as the set is never used before.

Reply:
It must be empty each time a new name server is inspected. It is defined as a "global variable" but it is used as a "local variable".

The two "empty" statements could be put just after the "for each child named server" if you think that is better.

Reply:
Same comment as above. *Has DNSKEY RRSIG Match DS* set is also used in step 16.

Reply:
I will investigate.